### ISSUE TITLE
Derivatives not indexing in Fedora 6 correctly

### ISSUE THREAD

**Issue by seth-shaw-unlv** [Islandora/documentation#1861]
For the Testing Sprint I created an audio item with a WMA file (I had to change the acceptable extensions). That saved to Fedora just fine. The derivative MP3 was also created and associated with the object as expected:
<img width="784" alt="Screen Shot 2021-06-25 at 2 10 27 PM" src="https://user-images.githubusercontent.com/29869988/123485529-261f7e00-d5bf-11eb-8569-5097b3bb26fb.png">

However, when I clicked on the derivative's Fedora URI I received the error `Error: No OCFL mapping found for info:fedora/ec/5b/79/0b/ec5b790b-d1e1-4d36-a143-54e4c2be1833/fcr:metadata`.

This happened with two separate attempts. I then attempted an Image and it also isn't working although the content that was loaded as part of `make demo` is appearing in Fedora just fine.

**Comment by dannylamb** [comment]
Confirmed, just with derivatives. Not sure what's going on here but I'll start looking at the logs.

**Comment by dannylamb** [comment]
Milliner logs are full of errors, but I think this is the source error here. 

```
milliner_1     | 2021-06-28T12:48:25.675874522Z [2021-06-28 12:48:25] app.DEBUG: > POST /external/14adce7e-a5b8-43b6-af9a-d20353a415b7 [] []
milliner_1     | 2021-06-28T12:48:25.712682774Z [2021-06-28 12:48:25] app.ERROR:  {"Exception":"[object] (RuntimeException(code: 400): Client error: `PUT http://islandora.traefik.me:8081/fcrepo/rest/14/ad/ce/7e/14adce7e-a5b8-43b6-af9a-d20353a415b7` resulted in a `400 Bad Request` response: Path did not match any allowed external content paths: https://islandora.traefik.me/sites/default/files/2021-06/40-FITS%20File.xml at /var/www/crayfish/Milliner/src/Service/MillinerService.php:498)"} []
```

This should be an easy fix.  :crossed_fingers:

**Comment by dannylamb** [comment]
It's getting an allowed external content url with `http` and not `https`.  Somehow the actual value supplied isn't getting  used and it's relying on an erroneous default value. 

In my `docker-compose.yml` I've got

```
  fcrepo:
    environment:
      FCREPO_ALLOW_EXTERNAL_DRUPAL: https://islandora.traefik.me/
```

But on the container I've got

```
$ printenv | grep DRUPAL
FCREPO_ALLOW_EXTERNAL_DRUPAL=http://islandora.traefik.me/
```

I need to update the default value to use https and also figure out why the supplied value is not getting used.
